src/web_app_skeleton/src/actions/errors/show.cr.ecr

```diff
@@ -20,6 +20,41 @@ class Errors::Show < Lucky::ErrorAction
     <%- end -%>
   end
 
+  def handle_error(error : Lucky::Exceptions::MissingNestedParam)
+    <%- if api_only? -%>
+    json Errors::ShowSerializer.new("Missing param key: '#{error.nested_key}'"), status: 400
+    <%- else -%>
+    if json?
+      json Errors::ShowSerializer.new("Missing param key #{error.nested_key}"), status: 400
+    else
+      render_unhandled_error(error)
+    end
+    <%- end -%>
+  end
+
+  # When an operation fails to save return the validation message
+  def handle_error(error : Avram::InvalidSaveOperationError)
+    <%- if api_only? -%>
+    render_invalid_param_json(error)
+    <%- else -%>
+    if json?
+      render_invalid_param_json(error)
+    else
+      render_unhandled_error(error)
+    end
+    <%- end -%>
+  end
+
+  private def render_invalid_param_json(error)
+    param = error.save_operation_errors.keys.first.to_s
+    serialized_error = Errors::ShowSerializer.new(
+      "Invalid params",
+      param: param,
+      details: "#{param} #{error.save_operation_errors.values.first.first}"
```

Comment:
I'm gonna add some stuff to Avram  to make this clearer. Basically it is getting the first validation error for the first attribute with an error